### PR TITLE
add latest hotspots tab

### PR DESCRIPTION
**Pull request by allenan**

### CODE REVIEW

File: data/hotspots.js [hunk 1]
```diff
@@ -71,3 +71,35 @@ export const fetchWitnesses = async (address) => {
   const witnesses = await list.take(TAKE_MAX)
   return witnesses
 }
+
+export const useHotspots = (pageSize = 20) => {
```
Comment (allenan) [inline data/hotspots.js:75]: there's a pattern developing around this infinite-pagination hook. will look to refactor as more examples come along

Comment (danielcolinjames) [reply data/hotspots.js:75]: 👍  nice yeah sounds good

Comment (danielcolinjames) [reply data/hotspots.js:75]: I actually had a sort of generic version of this in /data/activity.js over here: https://github.com/helium/explorer/blob/beta-blocks/data/activity.js#L37-L81

but probably not the best abstraction / naming / place for it to live

**Review by danielcolinjames — APPROVED** [review]
looks great! and with the flags this is a really cool way to see where new coverage is popping up in the immediate past

**Pull request merged**